test-application/parent.fyre

```diff
@@ -11,27 +11,32 @@ def ssr_request():
 
 <pyml>
     <span>
-    
+          
       <div>
         <h1> THIS IS MY FIRST HTML NODE </h1>
-        {ssr_request()} 
-        <p> THIS IS MY SECOND HTML NODE </p>
         {ssr_request()}         
+        <p> THIS IS MY SECOND HTML NODE </p>
+        {ssr_request()}    
+            
       </div>
       <b>      
         {use_parent_signal()}
+
       </b>
       <b>
         {get_parent_signal()}
       </b>
       <ul>
         <li>a</li>
+        
         <li>b</li>
         <li>c</li>
+        
       </ul>
       <div> 
         This won't be re-rendered
       </div>
+      <slot></slot>
```

Comment:
Hey @SuelenKarbivnychyy 👋 

Thank you for the PR 😄 

Just one question before I start the review - Is the content inside the `<slot>` tag used as a fallback?

Reply:
> Hey @SuelenKarbivnychyy 👋
> 
> Thank you for the PR 😄
> 
> Just one question before I start the review - Is the content inside the `<slot>` tag used as a fallback?

No. The fallback would be further implementation. I would suggest us to open an issue for that, and I will be happy to continue to implement it once this piece is merged :)